### ISSUE TITLE
Issue with sscanf

### ISSUE THREAD

**Issue by aristath** [scrutinizer-ci/php-analyzer#771]
This is the code I'm using:
```php
sscanf( $value, 'rgba(%d,%d,%d,%f)', $red, $green, $blue, $alpha );
```
The error I'm getting is that `$red`, `$greeen`, `$blue` and `$alpha` don't exist.
[`sscanf()` actually creates these, it does not use them](http://php.net/manual/en/function.sscanf.php) so this is not valid.

see https://scrutinizer-ci.com/g/aristath/kirki/inspections/9996cfab-e8cb-4a25-810f-91190caa00cd/issues/files/includes/class-kirki-sanitize-values.php?status=existing&orderField=path&order=asc&issueId=9a65e9be-8b65-4f2d-beaf-a75ed6043c37
